astyanax-cassandra/src/main/java/com/netflix/astyanax/serializers/JacksonSerializer.java

```diff
@@ -3,15 +3,17 @@
 import java.io.ByteArrayInputStream;
 import java.nio.ByteBuffer;
 
+import org.codehaus.jackson.annotate.JsonTypeInfo;
 import org.codehaus.jackson.map.ObjectMapper;
 import org.codehaus.jackson.map.annotate.JsonSerialize;
 
 public class JacksonSerializer<T> extends AbstractSerializer<T> {
+
+    @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
```

Comment:
please provide more context on this change

Reply:
I found this solution on [StackOverflow](https://stackoverflow.com/questions/58302380/how-to-i-disable-defaulttyping-is-it-disabled-by-default). But, I indicated the “type” property instead of the “className” because I applied it to the local variable

Reply:
this fix was canceled